test/cmdlineTests.sh

```diff
@@ -392,26 +392,9 @@ SOLTMPDIR=$(mktemp -d)
     cd "$SOLTMPDIR"
     "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/test/
     "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/docs/ docs
-    for f in *.sol
-    do
-        set +e
-        "$REPO_ROOT"/build/test/tools/solfuzzer --quiet < "$f"
-        if [ $? -ne 0 ]
-        then
-            printError "Fuzzer failed on:"
-            cat "$f"
-            exit 1
-        fi
 
-        "$REPO_ROOT"/build/test/tools/solfuzzer --without-optimizer --quiet < "$f"
-        if [ $? -ne 0 ]
-        then
-            printError "Fuzzer (without optimizer) failed on:"
-            cat "$f"
-            exit 1
-        fi
-        set -e
-    done
+    echo *.sol | xargs -P 10 -n 50 "$REPO_ROOT"/build/test/tools/solfuzzer --quiet --input-files
```

Comment:
I don't think `-P 10` is a good idea. This is usually run with at least one core at max already. I don't think we can assume more than 4 cores in most environments. I think the biggest speedup we get here is due to the `-n 50`.